content/tutorials/drift-detection/index.md

```diff
@@ -0,0 +1,198 @@
+{{% /notes %}}
+
+Click **Save deployment configuration**.
+
+![Animation of how to configure drift detection in Pulumi](./pulumi-configure-drift.mp4)
+
+Once you have configured Pulumi Deployments, open the Actions menu and choose `Detect Drift`, then click `Deploy` to validate your stack, program and deployments are all running before we introduce drift.
+
+![Screenshot of the Pulumi Cloud console to run a Drift Detection](./pulumi-detect-drift-action.png)
+
+Once the drift run is complete, you should see a new Drift check on the stack `Drift` Tab
+
+## Introduce drift
+
+To see Drift Detection in action, you will now introduce drift by manually making changes to your newly created EC2 resource, in this case by adding a new tag.
+
+Log in to the AWS Management console with your browser and ensure you are in the correct AWS region where your Pulumi stack is deployed.
+
+Modify your EC2 Instance by going to the EC2 Dashboard, then select the EC2 instance named `webserver` and click on the **Tags** tab. Next, add a tag by clicking the **Manage tags** button. Then click the **Add new tag** button with the key `Description` and value of `Pulumi Drift Detection`. Finally, **Save** the changes.
+
+## Run a Drift Detection deployment
+
+Return to the Pulumi Cloud and your stack, and again choose the **Detect Drift** action and click **Deploy**.
+
+After your run completes you will see a warning icon on the Drift tab, indicating that drift has been detected. You can see a detailed summary of what resources have been updated or deleted, the properties that have changed, when the drift run happened and a link to the Deployment with further details.
+
+![Animation of the Pulumi Cloud console Drift tab](./pulumi-drift-detected.mp4)
+
+## Remediate Drift
+
+Now that you have detected and reviewed your infrastructure drift, you can choose to remediate and overwrite the changes made in your cloud provider with the most recently specified desired state of your Pulumi program. This can be done via the **Remediate drift** option in the **Actions** menu on your stack page.
+
+Alternatively, you can accept the changes made in your cloud provider and apply them to your Pulumi state. This can be done via the **Refresh** option in the **Actions** drop down on your stack page. You will also want to update your Pulumi program to align with the changes made in the cloud provider.
+
+As the final step of this tutorial select the **Remediate drift** option in the Actions drop down on your stack page to remediate the manual changes. Once the deployment is complete, your infrastructure will be restored to the desired state expressed in your code.
```

Comment:
I do think this is missing something which is talking about how you can create drift detection on a schedule. Its a very easy mention that doesn't cost much time or space

Reply:
Added a section on this, but also plan to cover this in the next drift tutorial on Automation and notification, where i'll map out all the steps.